packages/backend/src/config/parseConfig.ts

```diff
@@ -330,6 +331,7 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
                 oauth2ClientSecret: process.env.AUTH_OKTA_OAUTH_CLIENT_SECRET,
                 authorizationServerId:
                     process.env.AUTH_OKTA_AUTHORIZATION_SERVER_ID,
+                extraScopes: process.env.AUTH_OKTA_EXTRA_SCOPES?.split(' '),
```

Comment:
we receive as a string, parse as an array of strings, and concat it again in `authentication.ts` - unnecessary or? could just be always a string 👍

Reply:
I would leave as a string!